contributors/new-contributor-playground/new-contributor-nodes.md

```diff
@@ -0,0 +1,346 @@
+# New kubernetes contributor workshop
+
+Joining in the beginning was onboarding on a yacht
+Now is more onboarding a BIG --Paquebot
+
+Will be a Hard schedule, and let's hope we can achieve everything
+Sig-contributor-experience -> from Non-member contributores to Owner
+
+## SIG presentation
+
+- SIG-docs & SIG-contributor-experience: **Docs and website** contribution
+- SIG-testing: **Testing** contribution
+- SIG-\* (*depends on the area to contribute on*): **Code** contribution
+
+**=> Find your first topics**: bug, feature, learning, community development and documentation
+
+Table exercise: Introduce yourself and give a tip on where you want to contribute in Kubernetes
+
+
+## Communication in the community
+
+Kubernetes community is like a Capybara: community members are really cool with everyone and they are from a lot of diffrenet horizons.
+
+- Tech question on Slack and Stack Overflow, not on Github
+- A lot of discussion will be involve when GH issues and PR are opened. Don't be frustrated
+- Stay patient because there is a lot of contribution
+
+When in doubt, **ask on Slack**
+
+Other communication channels:
+
+- Community meetings
+- Mailing lists
+- @ on Github
+- Office Hour
+- Kubernetes meetups https://www.meetup.com/topics/kubernetes
+
+on https://kubernetes.io/community,  there is the schedule for all the SIG/Working group meeting.
+If you want to join or create a meetup. Go to **slack#sig-contribex**
+
+## SIG - Special Interest Group
+
+Semi-autonomous teams:
+- Own leaders & charteers
+- Code, Github repo, Slack, mailing, meeting responsibility
+
+### Types
+
+[SIG List](https://github.com/kubernetes/community/blob/master/sig-list.md)
+
+1. Features Area
+    - sig-auth
+    - sig-apps
+    - sig-autoscaling
+    - sig-big-data
+    - sig-cli
+    - sig-multicluster
+    - sig-network
+    - sig-node
+    - sig-scalability
+    - sig-scheduling
+    - sig-service-catalog
+    - sig-storage
+    - sig-ui
+2. Plumbing
+    - sig-cluster-lifecycle
+    - sig-api-machinary
+    - sig-instrumentation
+3. Cloud Providers *(currently working on moving cloudprovider code out of Core)*
+    - sig-aws
+    - sig-azure
+    - sig-gcp
+    - sig-ibmcloud
+    - sig-openstack
+4. Meta
+    - sig-architecture: For all general architectural decision
+    - sig-contributor-experience: Helping contributor and community experience
+    - sig-product-management: Long-term decision
+    - sig-release
+    - sig-testing: In charge of all the test for Kubernetes
+5. Docs
+    - sig-docs: for documentation and website
+
+## Working groups and Subproject
+
+From working group to subproject. 
+
+for stuff really specific. Eg. `Helm` Belong to *SIG-apps* but dedicated subporject
```

Comment:
"subproject"

Reply:
I would change this text to:
```
For specific: tools (ex. Helm), goals (ex. Resource Management) or areas (ex. Machine Learning).
```

Reply:
I would also add that groups change around more frequently than SIGs, and some might be temporary.

Reply:
Thanks, updated